shell/client/shell.html

```diff
@@ -1287,8 +1324,6 @@ <h3 id="app-stats">Apps</h3>
   <form id="admin-settings-form">
     <p>Server Title: <input type="text" name="serverTitle" value="{{serverTitle}}"><br>
       <span class="details">The name of this server, for use when communicating with users. Example: "Bob's Sandstorm Server"</span></p>
-    <p>Return Address: <input type="email" name="returnAddress" value="{{returnAddress}}"><br>
```

Comment:
Server Title and Return Address are closely-related: the title is used as the display name in the "from" of emails sent. Today that's actually the only place it's used. Kind of sad for these now to be on separate pages.